src/csv/rowsReader/rowsReader.go

```diff
@@ -80,42 +80,23 @@ func (r *CsvReader) Err() error {
 	return r.scanner.Err()
 }
 
+// Search for \n -> rows delimiter. \n between enclosures is ignored.
 func splitRowsFunc(data []byte, atEOF bool) (advance int, token []byte, err error) {
 	length := len(data)
 
 	// Iterate over each character
-	index := 0
 	insideEnclosure := false
-	for index < length {
-		switch data[index] {
+	for index, char := range data {
+		switch char {
 		case CsvLineBreak:
 			if !insideEnclosure {
 				// Line break outside enclosure -> row delimiter, return row
 				return index + 1, data[0 : index+1], nil
 			}
 		case CsvEnclosure:
-			// We need to check next char, if 2 enclosures in a row -> then it is escaped enclosure -> skip
```

Comment:
Tak mi dnes doslo, ze toto je tu uplne zbytocne ... pri deleni na riadky nam staci ignorovat `\n` medzi dvoma `"` to je vsetko. Vsetky testy prechadzaju.

Je to teraz o par % cele rychlejsie.